stdlib/LinearAlgebra/src/generic.jl

```diff
@@ -1112,6 +1112,30 @@ function (\)(A::AbstractMatrix, B::AbstractVecOrMat)
 end
 
 (\)(a::AbstractVector, b::AbstractArray) = pinv(a) * b
+"""
+    A / B
+
+Matrix right-division: `A / B` is equivalent to `(B' \ A')'` where [`\`](@ref) is the left-division operator.
```

Comment:
`\`'s need to be escaped:
```suggestion
Matrix right-division: `A / B` is equivalent to `(B' \\ A')'` where [`\\`](@ref) is the left-division operator.
```